### PR TITLE
64492 chip lib error mapping using custom middleware

### PR DESCRIPTION
**Pull request by kanchanasuriya**
## Summary

- *Implemented middleware for chip service library to handling the error scenarios*

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#64492

## Testing done
- [x] rspec
- [x] rubocop

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*
NA

## Acceptance criteria

- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by rmtolmach** [comment]
> Error: A file (or its parent directories) does not have a CODEOWNERS entry. Please update the .github/CODEOWNERS file and add the entry for the Offending file: config/locales/exceptions.en.yml

@kanchanasuriya Can you add this line to [CODEOWNERS](https://github.com/department-of-veterans-affairs/vets-api/blob/master/.github/CODEOWNERS#L773) (around line 773)?
```CODEOWNERS
config/locales @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group 
```

**Comment by kanchanasuriya** [comment]
> > Error: A file (or its parent directories) does not have a CODEOWNERS entry. Please update the .github/CODEOWNERS file and add the entry for the Offending file: config/locales/exceptions.en.yml
> 
> @kanchanasuriya Can you add this line to [CODEOWNERS](https://github.com/department-of-veterans-affairs/vets-api/blob/master/.github/CODEOWNERS#L773) (around line 773)?
> 
> ```
> config/locales @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group 
> ```

Updated CODEOWNERS file for config/locale

**Review by randomsync — APPROVED** [review]
lgtm

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1900,6 +1901,7 @@ spec/support/vcr_cassettes/veteran_readiness_employment @department-of-veterans-
 spec/support/vcr_cassettes/vetext @department-of-veterans-affairs/mobile-api-team
 spec/support/vcr_cassettes/virtual_agent @department-of-veterans-affairs/vfs-virtual-agent-chatbot @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 spec/support/vcr_cassettes/virtual_regional_office @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+spec/support/vcr_cassettes/chip/ @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```
Comment (RachalCassity) [inline .github/CODEOWNERS:1904]: Please add the correct team that owns this folder.

**Review by jperk51 — APPROVED** [review]
mobile changes look good

**Review by RachalCassity — APPROVED** [review]

**Pull request merged**